doc/nrf/known_issues.rst

```diff
@@ -1873,7 +1869,7 @@ SoftDevice Controller
 
 In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
 
```

Comment:
Add the below softdevice controller issues from https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/softdevice_controller/CHANGELOG.html#bug-fixes as known issues  (even though all have been fixed in NCS 2.1):

.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
DRGN-17656: HardFault with periodic advertising synchronization
  When creating a periodic advertising synchronization, a HardFault might occur if receiving legacy advertising PDUs.

.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0
DRGN-17454: Wrong data length is selected if the even length is greater than 65535 us
  If the event length (CONFIG_BT_CTLR_SDC_MAX_CONN_EVENT_LEN_DEFAULT) was set to a value greater than 65535 us, the auto-selected maximum data length was set to 27 bytes due to an integer overflow.

.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0
DRGN-17651: Incorrect memory usage if configuring fewer TX/RX buffers than the default value
  When using the memory macros with less TX/RX packet count than the default value, the actual memory usage might be higher than expected.

.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
DRGN-17710: Periodic advertiser sends AUX_SYNC_IND 40 us later than indicated in the SyncInfo of the AUX_ADV_IND
  The periodic advertiser sends its packet 40 us later than the one indicated in the SyncInfo of the AUX_ADV_IND packet.

.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
DRGN-17710: The scanner would attempt to receive the first AUX_SYNC_IND 40 us later than the one indicated in the SyncInfo of the AUX_ADV_IND
  This may result in the device failing to establish a synchronization to the periodic advertiser.

Reply:
Added